packages/webapp/index.html

```diff
@@ -19,20 +19,15 @@
     <link rel="stylesheet" href="/global.css?v=1.0.3" />
 
     <link rel="manifest" href="/manifest.json" />
-    <link rel="apple-touch-icon" sizes="57x57" href="/apple-icon-57x57.png" />
-    <link rel="apple-touch-icon" sizes="60x60" href="/apple-icon-60x60.png" />
-    <link rel="apple-touch-icon" sizes="72x72" href="/apple-icon-72x72.png" />
-    <link rel="apple-touch-icon" sizes="76x76" href="/apple-icon-76x76.png" />
-    <link rel="apple-touch-icon" sizes="114x114" href="/apple-icon-114x114.png" />
-    <link rel="apple-touch-icon" sizes="120x120" href="/apple-icon-120x120.png" />
-    <link rel="apple-touch-icon" sizes="144x144" href="/apple-icon-144x144.png" />
-    <link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png" />
-    <link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png" />
-    <link rel="icon" type="image/png" sizes="192x192" href="/android-icon-192x192.png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
```

Comment:
Is there a tool you use to keep these updated and know which are necessary? If yes can we put a comment somewhere in here with last updated using X tool sort of thing?

Reply:
https://vite-pwa-org.netlify.app/guide/pwa-minimal-requirements.html#icons-images
Everything is in vite pwa doc